products/jhipster.md

```diff
@@ -0,0 +1,75 @@
+---
+title: JHipster
+category: app
+versionCommand: yo jhipster
+permalink: /jhipster
+releasePolicyLink: https://www.jhipster.tech/releases/
+changelogTemplate: "https://github.com/jhipster/generator-jhipster/tag/__LATEST__"
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Security Support
+#iconSlug: jhipster , WIP see https://github.com/jhipster/generator-jhipster/issues/20533
+
+#auto:
+#-   maven: org.apache.maven/maven-core
```

Comment:
Could we use https://www.npmjs.com/package/generator-jhipster?activeTab=versions perhaps? 

(Once https://github.com/endoflife-date/release-data/pull/36 is merged, ie)